--- build/buildcheck.sh.ok	2018-01-24 13:55:10.602429830 +0100
+++ build/buildcheck.sh	2018-01-24 13:55:47.662322378 +0100
@@ -28,18 +28,18 @@
   PHP_AUTOCONF='autoconf'
 fi
 
-# autoconf 2.64 or newer
+# autoconf 2.63 or newer
 ac_version=`$PHP_AUTOCONF --version 2>/dev/null|head -n 1|sed -e 's/^[^0-9]*//' -e 's/[a-z]* *$//'`
 if test -z "$ac_version"; then
 echo "buildconf: autoconf not found."
-echo "           You need autoconf version 2.64 or newer installed"
+echo "           You need autoconf version 2.63 or newer installed"
 echo "           to build PHP from Git."
 exit 1
 fi
 IFS=.; set $ac_version; IFS=' '
-if test "$1" = "2" -a "$2" -lt "64" || test "$1" -lt "2"; then
+if test "$1" = "2" -a "$2" -lt "63" || test "$1" -lt "2"; then
 echo "buildconf: autoconf version $ac_version found."
-echo "           You need autoconf version 2.64 or newer installed"
+echo "           You need autoconf version 2.63 or newer installed"
 echo "           to build PHP from Git."
 exit 1
 else
